src/scripts/main.js

```diff
@@ -1,3 +1,216 @@
 'use strict';
 
 // write code here
+
+// Сортування таблиці
+
+const table = document.querySelector('table');
+const tbody = document.querySelector('tbody');
+const tableHeader = Array.from(table.querySelectorAll('th'));
+
+const sortDirection = {};
+
+function parseNumber(value) {
+  return parseFloat(value.replace(/[^0-9.]/g), '');
+}
+
+tableHeader.forEach((header, columIndex) => {
+  header.addEventListener('click', () => {
+    // Визначення напрямку сортування
+    sortDirection[columIndex] = !sortDirection[columIndex];
+
+    // Вибираємо всі рядки tbody
+    const rows = Array.from(tbody.querySelectorAll('tr'));
+
+    // Сортуємо рядки таблиці
+    rows.sort((a, b) => {
+      const cellA = a.children[columIndex].textContent;
+      const cellB = b.children[columIndex].textContent;
+
+      const numberA = parseNumber(cellA);
+      const numberB = parseNumber(cellB);
+
+      // Перевіряємо чи рядок є яислом
+      if (!isNaN(numberA) && !isNaN(numberB)) {
+        return sortDirection[columIndex]
+          ? numberA - numberB
+          : numberB - numberA;
+      } else {
+        return sortDirection[columIndex]
+          ? cellA.localeCompare(cellB)
+          : cellB.localeCompare(cellA);
+      }
+    });
+
+    tbody.innerHTML = '';
+    // Додаємо відсортовані елементи в tbody
+    rows.forEach((row) => tbody.appendChild(row));
+  });
+});
+
+// Active row
+
+tbody.addEventListener('click', (e) => {
+  const clikedRow = e.target.closest('tr');
+
+  if (clikedRow) {
```

Comment:
The variable `clikedRow` should be corrected to `clickedRow` to match the corrected variable name.